src/Core/Settings.cpp

```diff
@@ -5560,6 +5560,9 @@ Only available in ClickHouse Cloud. Exclude new data parts from SELECT queries u
 )", 0) \
     DECLARE(Int64, prefer_warmed_unmerged_parts_seconds, 0, R"(
 Only available in ClickHouse Cloud. If a merged part is less than this many seconds old and is not pre-warmed (see cache_populated_by_fetch), but all its source parts are available and pre-warmed, SELECT queries will read from those parts instead. Only for ReplicatedMergeTree. Note that this only checks whether CacheWarmer processed the part; if the part was fetched into cache by something else, it'll still be considered cold until CacheWarmer gets to it; if it was warmed, then evicted from cache, it'll still be considered warm.
+)", 0) \
+    DECLARE(Bool, allow_experimental_database_iceberg, false, R"(
```

Comment:
The setting should use the `EXPERIMENTAL` flag (instead of 0) and ideally it should be in the experimental block (at the end of the list)

Reply:
Fixed in https://github.com/ClickHouse/ClickHouse/pull/73262